### DISCUSSION TITLE
IBEX workflow graphics?

### DISCUSSION THREAD

**Discussion by alunich** [IBEXImagingCommunity/ibex_imaging_knowledge_base#128] [Q&A] [answered]
Hi everyone! 
I am working on a poster that I plan to present on my IBEX work next month and was wondering if anyone had any IBEX workflow graphics for me to use that they would be willing to share.

Thanks!

**Comment by radtkea** [comment, marked as answer]
Hi Alunich,

Warmest congratulations on your poster presentation! For the workflow, I generally grab Figure 1 from our _Nature Protocols_ article: https://www.nature.com/articles/s41596-021-00644-9/figures/1

You can crop and focus on the method and registration workflows as I have shown here. 
![Overall protocol Final-600dpi signature-082021-01](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/cd98b5a1-9380-4f34-a267-34c0a645c58f)

Another option is to use the IBEX workflow found in Supplementary Figure 3 of our _PNAS_ paper attached here.
![Figure S3](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/ccbfcd6d-2552-4b59-b7da-4be0ce29a263)

Don't forget to include the QR code in your poster: https://raw.githubusercontent.com/IBEXImagingCommunity/ibex_imaging_large_files/main/ibex_logos/IBEX_QR.png

Hope this helps! Best of luck with your presentation:)
A

**Reply by alunich** [reply]
Hi A, 

Thank you so much for your quick reply! These images are exactly what I needed.

**Reply by radtkea** [reply]
Awesome, so happy to hear that! Best of luck with your poster presentation. Please share a picture of you with your poster, if you feel like it. Best wishes!
